### ISSUE TITLE
[FormRecognizer] Remove IgnoreServiceError attribute for "ContentSourceNotAccessible"

### ISSUE THREAD

**Issue by kinelski** [Azure/azure-sdk-for-net#28923]
## Error information

Error code: ContentSourceNotAccessible
Error message: "Content is not accessible: Invalid data URL"
When: when starting a model-building LRO.
API version: v2022-01-30-preview
apim-request-id: 7d771404-5269-4099-9e51-e8ba8ca0b4d2
Example: https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1575344&view=logs&j=01d5a77e-6e85-5280-0e9f-af5629bd443f&t=1e514895-521a-57d8-4030-b9d8f4f46b20&l=272

## Additional details

This is a flaky service error that doesn't happen often, but it's frequent enough to happen on almost every test pipeline run. The service team has already been notified about it ([discussion here](https://teams.microsoft.com/l/message/19:88f2d9dea2344075b5d4bc34d82a3d1c@thread.skype/1652910631452?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1652910631452&teamName=Azure%20SDK&channelName=Service%20-%20Form%20Recognizer&createdTime=1652910631452)).

This error won't make our pipeline fail anymore since it's been suppressed with the `IgnoreServiceError` attribute in our test project. The attribute is set on the class level (instead of single method) because it can happen on any test that builds a model, which includes most of our tests.

## Action items

Once the service has fixed this issue on their side, we must remove the `IgnoreServiceError` attribute from the following classes:
- `DocumentModelAdministrationClientLiveTests`

**Comment by prodramp** [comment]
I am hitting the same error when I build the model with V3.0 custom model. When I access the same PDF/JSON documents from V2.1 the model does build however I want to use Table layout so V3.0 is most appropriate for me. 
When I can expect the fix of this issue? Are there any workaround to build my model? 

Thanks

**Comment by kinelski** [comment]
@prodramp The bug described in this issue does not occur frequently and only happens when we put the service under a heavy load of requests. My understanding is that you were seeing it every time you try to build a model, is that right? In that case it may have a different cause.

I'm closing this issue because it will be merged into https://github.com/Azure/azure-sdk-for-net/issues/31914, but feel free to continue the discussion there if you are still blocked by it.
